packages/create-next-app/templates/app/js/app/globals.css

```
@@ -86,7 +86,7 @@ body {
 }
 
 body {
-  color: rbg(--foreground-rgb);
+  color: rbg(var(--foreground-rgb));
```

It should be `rgb` right?

```suggestion
  color: rgb(var(--foreground-rgb));
```

---

I...apologize to Ruth Bader Ginsburg for misusing her initials -- fix pushed! @shuding